### PR TITLE
activate the Create Adpool button for everyone

### PR DESCRIPTION
**Pull request by raniamorheg**
##  Description:
Hey team,

This PR focuses on activating the "Create Adpool" button for all users in our application. Currently, the button is not available to everyone, causing inconvenience. To improve the user experience, we have made the following changes:

Enabled the "Create Adpool" button for all users, ensuring universal access and convenience.
Additionally, we have implemented the "createNewDraftCampaign()" function to enhance the functionality of the ad creation process. This function enables users to easily initiate a new draft campaign and streamline their workflow.

Implemented the "createNewDraftCampaign()" function to facilitate the creation of new draft campaigns, allowing users to initiate campaigns seamlessly.
Furthermore, we have made design modifications to improve the visual aesthetics and user interface of the application. These changes enhance the overall user experience and align with modern design standards.

Modified the design elements to enhance visual aesthetics and improve the user interface, creating a more enjoyable user 

##  Notes for Reviewers:

Kindly review the changes made to ensure the "Create Adpool" button is activated for all users.
Please validate the implementation of the "createNewDraftCampaign()" function to verify its functionality and adherence to best practices.
Review the design modifications to ensure they align with the desired visual aesthetics and enhance the user interface.
We value your feedback, suggestions, and alternative approaches to further improve the implementation and user experience.

Thank you for your time and consideration. Your feedback is highly appreciated.

Best regards,
Rania Morheg

### CODE REVIEW

**Comment by raniamorheg** [comment]
##Description:
Hey team,

This PR focuses on activating the "Create Adpool" button for all users in our application. Currently, the button is not available to everyone, causing inconvenience. To improve the user experience, we have made the following changes:

Enabled the "Create Adpool" button for all users, ensuring universal access and convenience.
Additionally, we have implemented the "createNewDraftCampaign()" function to enhance the functionality of the ad creation process. This function enables users to easily initiate a new draft campaign and streamline their workflow.

Implemented the "createNewDraftCampaign()" function to facilitate the creation of new draft campaigns, allowing users to initiate campaigns seamlessly.
Furthermore, we have made design modifications to improve the visual aesthetics and user interface of the application. These changes enhance the overall user experience and align with modern design standards.

Modified the design elements to enhance visual aesthetics and improve the user interface, creating a more enjoyable user experience.
##Notes for Reviewers:

Kindly review the changes made to ensure the "Create Adpool" button is activated for all users.
Please validate the implementation of the "createNewDraftCampaign()" function to verify its functionality and adherence to best practices.
Review the design modifications to ensure they align with the desired visual aesthetics and enhance the user interface.
We value your feedback, suggestions, and alternative approaches to further improve the implementation and user experience.

Thank you for your time and consideration. Your feedback is highly appreciated.

Best regards,
Rania Morheg

**Comment by raniamorheg** [comment]
##  Description:
Hey team,

This PR focuses on activating the "Create Adpool" button for all users in our application. Currently, the button is not available to everyone, causing inconvenience. To improve the user experience, we have made the following changes:

Enabled the "Create Adpool" button for all users, ensuring universal access and convenience.
Additionally, we have implemented the "createNewDraftCampaign()" function to enhance the functionality of the ad creation process. This function enables users to easily initiate a new draft campaign and streamline their workflow.

Implemented the "createNewDraftCampaign()" function to facilitate the creation of new draft campaigns, allowing users to initiate campaigns seamlessly.
Furthermore, we have made design modifications to improve the visual aesthetics and user interface of the application. These changes enhance the overall user experience and align with modern design standards.

Modified the design elements to enhance visual aesthetics and improve the user interface, creating a more enjoyable user ## ## Notes for Reviewers:

Kindly review the changes made to ensure the "Create Adpool" button is activated for all users.
Please validate the implementation of the "createNewDraftCampaign()" function to verify its functionality and adherence to best practices.
Review the design modifications to ensure they align with the desired visual aesthetics and enhance the user interface.
We value your feedback, suggestions, and alternative approaches to further improve the implementation and user experience.

Thank you for your time and consideration. Your feedback is highly appreciated.

Best regards,
Rania Morheg

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -415,6 +415,7 @@
   "AllCampains.Modify_campagin": "Finalize my Pool",
   "AllCampains.delete_camp_quest": "Are you sure you want to delete this Ad Pool?",
   "AllCampains.Confirm": "Confirm",
+  "add_pool":"Add your Pool",
```
Comment (ksibisamir) [inline src/assets/i18n/en.json]: Create your AdPool

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -231,6 +231,7 @@
   "Ajouter des liens": "Ajouter des liens",
   "Ajouter un lien": "Ajouter un lien",
   "All_campaign": "Toutes les campagnes",
+  "add_pool":"Creer Pool",
```
Comment (ksibisamir) [inline src/assets/i18n/fr.json]: Créer AdPool

Comment (ksibisamir) [reply src/assets/i18n/fr.json]: Merci de renommer la  variable par  "Create.Adpool"

**Review by ksibisamir — APPROVED** [review]

**Pull request merged**